pkgs/by-name/tr/trexio/package.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv
+, lib
+, fetchurl
+, cmake
+, gfortran
+, hdf5
+,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "trexio";
+  version = "2.4.2";
+
+  # TRex IO contains some generated code, which is part of the source tarball
```

Comment:
Isn't that just generated by some py script in the tools dirctory?

Reply:
Hm yes but as far as I understood the documentation I requires Emacs? Also, the `.devel` file needs to be removed and I thought if it it isn't necessary to depend on python, then why do it? :)

Reply:
Building from the github sources now.

Reply:
> Hm yes but as far as I understood the documentation I requires Emacs? 

That's a bit awkward, but the result looks nice and clean now.